worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1417,6 +1418,42 @@ public int stackCuboidRegion(Region region, BlockVector3 offset, int count,
         return copy.getAffected();
     }
 
+    /**
+     * Stack a cuboid region using block units.
```

Comment:
Maybe suggest using `getBoundingBox()`?

Reply:
I'm unsure if it makes sense to tbh, as that includes areas outside the region itself